cf-java-monitoring-custom-metrics-clients/cf-custom-metrics-clients-spring-boot/src/test/java/com/sap/cloud/cf/monitoring/spring/EnvUtils.java

```diff
@@ -0,0 +1,33 @@
+package com.sap.cloud.cf.monitoring.spring;
+
+import java.lang.reflect.Field;
+import java.util.Collections;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class EnvUtils {
```

Comment:
Can you provide this utility class centrally instead of maintaining two copies of the code?

Reply:
The test classes/packages are not packed and it could not be used as a dependency. If I want to reuse the class I should move the class in the productive packages which is not the idea.

Reply:
You can create a test-jar to reuse test code:

https://maven.apache.org/guides/mini/guide-attached-tests.html

Reply:
Thanks, looks good. We can use this (refactor) with the next version? I do not want to change other non-critical parts of the proposed change.

Reply:
If you want to wait until the next version, please add an issue so thatwe can track progress and don't forget to do it.

Reply:
Done